templates/whatson/whats_on_page.html

```diff
@@ -67,20 +67,9 @@ <h2 class="tna-heading tna-heading--l event-card-listing__heading">All events</h
         <p class="event-card-listing__summary">Showing 1 to 12 of {{ page.events|length }} events</p>
       </hgroup>
 
-      <ul class="event-card-listing__list">
-        {% if page.events %}
-          {% for event in page.events %}
-            <li class="event-card-listing__item">
-              {% image event.teaser_image fill-720x448 as event_img %}
-              {% include "../components/card.html" with title=event.short_title href=event.url headingLevel="2" headingSize="l" htmlElement="article" imageSrc=event_img.url imageAlt=event_image.width imageWidth=event_img.width imageHeight=event_img.height date=event.start_date|date:"jS F Y" cost=event.price_range venue_type=event.get_venue_type_display body=event.teaser_text horizontal=True event=True highlight="Sold out" highlight_color="yellow" event_type=event.event_type access_descriptor=event.primary_access_type classes="tna-card--event tna-card--horizontal-thirds" index=forloop.counter0 heading=heading %}
-            </li>
-          {% endfor %}
-        {% else %}
-          <li class="event-card-listing__item">
-            <p>There are no events to display.</p>
-          </li>
-        {% endif %}
-      </ul>
+      <div>
```

Comment:
This empty div has a data attribute in the filters merge request which is used for reloading the content of the listing.